spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -612,6 +614,9 @@ def __init__(self, parent=None):
         completion_parent = self.completion_widget.parent()
         self.kite_call_to_action = KiteCallToAction(self, completion_parent)
 
+        # Some events should not be triggered on undo/redo

```

Comment:
Which type of events? Could you extend a bit here or add examples so we now what those events are or could be in the future?

Reply:
see discussion about `cursor_changed`.

Reply:
👍